### PR TITLE
[Paddle Integration] Add Widgets for PaddleNLP

### PR DESCRIPTION
**Pull request by sijunhe**

### CODE REVIEW

File: js/src/lib/interfaces/Libraries.ts [hunk 1]
```diff
@@ -148,6 +149,28 @@ const keras = (model: ModelData) =>
...
+		].join("\n");
+	} else {
+		return [
+			`from paddlenlp.transformers import ${info.auto_model}`,
+			"",
+			`model = ${info.auto_model}.from_pretrained("${model.id}"${model.private ? ", use_auth_token=True" : ""}, from_hf_hub=True)`,
+		].join("\n");
+	}
+};
+
```
Comment (osanseviero) [inline js/src/lib/interfaces/Libraries.ts:173]: To simplify this PR as the information from the PaddleNLP is not in the `transformersInfo` as the config/library is different, would it be possible to do a very high-level snippet and afterward we could increase the coverage? First, we need this PR merged to start working on the internal side of things to add more info to the `ModelData`. Something along these lines


```suggestion
const paddlenlp = (model: ModelData) => {
	return [
	  `from paddlenlp.transformers import AutoModel`,
	  "",
	  `model = AutoModel.from_pretrained("${model.id}"${model.private ? ", use_auth_token=True" : ""}, from_hf_hub=True)`,
        ].join("\n");
};

```

**Review by osanseviero — COMMENTED** [review]
Thanks a lot for the PR

File: js/src/lib/interfaces/Libraries.ts (hunk 1 above)
Comment (sijunhe) [reply js/src/lib/interfaces/Libraries.ts]: Thanks for the suggestion. I basically took your suggestion. Only added a `AutoTokenizer`

**Review by osanseviero — APPROVED** [review]
Thanks a lot! 🔥

**Comment by sijunhe** [comment]
Thanks for the review @osanseviero. Are we merging this? I don't have write access so I can't merge

**Comment by osanseviero** [comment]
Yes, merged!

**Pull request merged**